client/src/component/Review.jsx

```diff
@@ -1,29 +1,124 @@
 import React from 'react';
+import styled from 'styled-components';
 
+const UserProfilePicture = styled.img`
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
+`;
+const UsernameStyle = styled.div`
+  font-weight: 600;
+  margin-bottom: 0px;
+  margin-top: 0px;
+  font-size: 16px;
+  font-family: Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif;
+  line-height: 1.375em;
+  color: #484848;
+`;
+const MessageStyle = styled.div`
+  color: #484848;
+  margin-top: 16px;
+`;
+const TimeAndUsernameContainer = styled.div`
+  margin-left: 16px;
+`;
+const TimeUsernameProfilePicContainer = styled.div`
+display: inline-flex;
+width: 100%;
+border-spacing: 0px;
+`;
+const TimeStyle = styled.span`
+margin: 0px;
+font-family: Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif;
+font-size: 14px;
+font-weight: 400;
+line-height: 1.2857142857142858em;
+color: #484848;
+`;
+const ReviewContainer = styled.div`
+  margin-bottom: 35px;
+  max-width: 696px;
+  margin-left: 70px;
+`;
+const ReadMore = styled.p`
+  color: #008489;
+  display: inline-flex;
+  margin: 0px
+`;
 class Review extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      readMoreClick: false,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event, review) {
+    event.preventDefault();
+    review.readMoreClicked = true;
+    this.setState({ readMoreClick: true });
   }
 
   render() {
+    const { readMoreClick } = this.state;
     const { review } = this.props;
     const {
-      profile_pic_url, username, time_made, message 
+      profile_pic_url, username, time_made, firstHalfOfMessage, secondHalfOfMessage, readMoreClicked,
     } = review;
-    return (
-      <div>
-        <div>
+    if (readMoreClick === true && readMoreClicked === true) {
+      return (
+        <ReviewContainer>
           <div>
-            <div><img src={profile_pic_url} alt="" /></div>
-            <div>{username}</div>
-            <div>{time_made}</div>
+            <TimeUsernameProfilePicContainer>
+              <UserProfilePicture src={profile_pic_url} alt="" />
+              <TimeAndUsernameContainer>
+                <div>
+                  <UsernameStyle>{username}</UsernameStyle>
```

Comment:
I believe convention is to call divs that have been styled using styled components as "Styled+restOfName".
```suggestion
                  <StyledUsername>{username}</StyledUsername>
```